### ISSUE TITLE
Sorting Parent Posts Dropdown

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#227]
```
When using custom hierarchy I need to set the post's parent. 
Specially when you have too many posible parents it is difficult to find the 
exact post I want because the dropdown is ordered by date. It would be nice if 
we could select the default sorting field and order. 

In my case I have solved the issue adding the following code inside the 
function "customized_hierarchical_post_types" in "StandadizedCustomFields.php":

                        $args['orderby'] = 'title';
                        $args['order'] = 'ASC';

The bad thing is I must keep an eye after updating the plugin. 

```

Original issue reported on code.google.com by `lsant...@gmail.com` on 28 Oct 2011 at 8:38

**Comment by GoogleCodeExporter** [comment]
```
Yes, I'm refactoring this by integrating my Summarize Posts plugin, which will 
let you specify all kinds of filters when you search for another post -- this 
is a good point.  The refactoring was focusing on the Relation fields (e.g. 
Image fields, media fields), but this should apply to the custom hierarchies as 
well.  

Related: issue 8.
```

Original comment by `fireproofsocks` on 28 Oct 2011 at 3:18
- Changed state: **Accepted**
- Added labels: **Milestone-Release0.9.5**, **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Basic fix Committed revision 457617.  Working on the high-octane version of 
this using the Summarize Posts plugin...
```

Original comment by `fireproofsocks` on 31 Oct 2011 at 5:12
- Changed state: **InProgress**

**Comment by GoogleCodeExporter** [comment]
```
Another suggestion, and actually asking for a possible suggestion back.

Let's imagine the following custom content types and hierarchy :
country > state > city

When I'm creating a new "city" I must choose it's parent "state" among all the 
"states" available in any "country".
It would be a better user experience if I could filter by choosing at first 
"country" and then a "state" among the "states" in the "country" I had 
specified before. In other words, choose a "post" parent from the top of the 
hierarchy till the bottom.

Actually it would be even better if I could use that kind of relation in any 
custom field. So wouldn't need to set a parent just a new "relation field".

The suggestion I ask is, do you know/recommend any CMS that does this kind of 
content type relations out of the box, or at least with some plugins. I love 
wordpress for it's simple interface , that makes easier for the regular user. 
But I lack such feature in some projects I am working on.








```

Original comment by `lsant...@gmail.com` on 9 Nov 2011 at 4:12

**Comment by GoogleCodeExporter** [comment]
```
Yes, it's surprising that so many people have suggested more or less the exact 
same thing as I've been working on it.  See issue 8 -- there dev version of 
0.9.5 that does this on the dev branch that lets you specify a lot of detail 
when you set up a relation field and when you make the selection when you 
create/edit the post.  I'm still weeding through the other fields for 0.9.5.  

See http://wordpress.org/extend/plugins/custom-content-type-manager/download/ 
for the dev version.

The initial sorting thing has been included in 0.9.5.
```

Original comment by `fireproofsocks` on 9 Nov 2011 at 5:35
- Changed state: **Fixed**
